changelogs/fragments/526-swap-tls-accept-connect-in-proxy.yaml

```
@@ -0,0 +1,2 @@
+bugfixes:
+  - zabbix_proxy - swap configuration of tls_accept and tls_connect
```

```suggestion
  - zabbix_proxy - correctly provision tls_accept and tls_connect on Zabbix backend
```
Since this will be visible to end user I think we need to re-word it a bit.